### ISSUE TITLE
IDEA GATHERING: Clicking on close button in folders makes the idea view disappear 

### ISSUE THREAD

**Issue by ssin1901** [citizenos/citizenos-fe#1704]
**What is the problem?**
_Tested in Google Chrome in incognito mode_

When a user clicks on folder in this topic: https://app.citizenos.com/en/topics/63fbb9ac-d9d1-40cb-a795-188fba61a7d2#ideation and clicks the close button, the idea view disappears (see screen recording)

https://github.com/user-attachments/assets/7e522289-713b-4913-b9c2-897f850d3696

**Why is this a problem?**
User loses the idea view until they refresh the page 

**Possible solution.**
@BeccaMelhuish @ilmartyrk if I'm not mistaken, the user should see this navigation tab after closing folders? (screenshot1)

<img width="785" alt="Screenshot 2024-11-29 at 00 25 45" src="https://github.com/user-attachments/assets/2532503e-47e2-454b-ac26-1e37e54e6410">

**Comment by BeccaMelhuish** [comment]
Ilmar: should be quick, should be done with URL to idea folder issue. Becca to find the issue and link them. 
Est: 3 hours for the 2 issues.

**Comment by BeccaMelhuish** [comment]
@ilmartyrk I tried searching for relevant key words and the only issue I could find was [this one](https://github.com/citizenos/citizenos-fe/issues/1285) for direct links to ideas rather than to idea folders. 

If I correctly understood the issue you had in mind, shall we make a new issue to add 'Share direct link' to the folder here?  

![Image](https://github.com/user-attachments/assets/e52186b2-84f2-47c0-8fb2-ba6b429942d4)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish yes, please create a new issue for the share direct link. It can have good-first-issue tag too

**Comment by BeccaMelhuish** [comment]
**Dev task summary:** 

Fix the bug shared by Sara above. Should be fixed in parallel with https://github.com/citizenos/citizenos-fe/issues/1728

**Comment by BeccaMelhuish** [comment]
Tested in dev and working well, I get this view after closing (on desktop and mobile) which is as it should be :) Moving to pre-live. 

![image](https://github.com/user-attachments/assets/9f0442aa-722c-49a7-ab5d-0196fe8bf68a)
